project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -3204,6 +3204,12 @@ function c4m_message__update__comment($comment) {
  *   The non-saved message entity.
  */
 function c4m_message__insert__node($node) {
+  // Do not create message for activity stream, as we deal with
+  // unpublished content.
+  if ($node->status == NODE_NOT_PUBLISHED) {
```

Comment:
`==` seems to be sufficient for node status check (went through code that checks for node status, and it's always `==`).